tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -420,6 +420,13 @@ internal static class FeatureFlags
             /// </summary>
             /// <seealso cref="TracerSettings.RouteTemplateResourceNamesEnabled"/>
             public const string RouteTemplateResourceNamesEnabled = "DD_TRACE_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED";
+
+            /// <summary>
+            /// Feature Flag: disables the replacement of periods by underscores in header tags.
+            /// This is needed for compatibility with other tracers
+            /// </summary>
+            /// <seealso cref="TracerSettings.LeavePeriodsInHeaderTags"/>
+            public const string LeavePeriodsInHeaderTags = "DD_TRACE_LEAVE_PERIODS_IN_HEADER_TAGS";
```

Comment:
The standard name for feature flags is `DD_TRACE_{0}_ENABLED`. 

I think we should probably call this `DD_TRACE_REPLACE_HEAD_TAG_PERIODS_ENABLED`. This moves from `leave` to `replace` because "replace" feels like a positive action which is preferable for booleans. The default would then be `true`, and customers can disable the flag. 

Not sure if everyone will agree with that though :D

Reply:
I knew my "Leave" would bring comments. Naming variables isn't my strong suit ;)
And indeed having the replace_enabled seems more straightforward